GVFS/GVFS.NativeTests/GVFS.NativeTests.vcxproj

```diff
@@ -14,21 +14,21 @@
     <ProjectGuid>{3771C555-B5C1-45E2-B8B7-2CEF1619CDC5}</ProjectGuid>
     <Keyword>Win32Proj</Keyword>
     <RootNamespace>GVFSNativeTests</RootNamespace>
-    <WindowsTargetPlatformVersion>10.0.10240.0</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion>10.0.16299.0</WindowsTargetPlatformVersion>
   </PropertyGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" />
   <Import Project="$(SolutionDir)\GVFS\GVFS.Build\ProjFS.props" />
   <Import Project="$(SolutionDir)\GVFS\GVFS.Build\GVFS.cpp.props" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x64'" Label="Configuration">
     <ConfigurationType>DynamicLibrary</ConfigurationType>
     <UseDebugLibraries>true</UseDebugLibraries>
-    <PlatformToolset>v141</PlatformToolset>
+    <PlatformToolset>v142</PlatformToolset>

```

Comment:
Changing from v141 to v142 will probably break VS2017 users, since I don't think it is available.
The alternative is to add the v141 package to your VS2019 installation.

I'm not sure how the build machines are configured.

Reply:
Oh, this commit will _definitely_ break 2017 users.

Reply:
Is there any impact on VS for Mac?

Reply:
You should be able to install the v141 build tools in VS 2019, under the "Desktop development with C++" components.

It might be a little soon to require everyone to update to VS 2019, but that is another option.